frontend/packages/core/src/AppProvider/index.tsx

```diff
@@ -25,6 +25,11 @@ export interface UserConfiguration {
   };
 }
 
+export interface AppConfiguration {
+  title?: string;
+  logo?: React.ReactNode;
```

Comment:
Really minor and we can do this later but just to discuss - do we want to support the ability to read this in from an asset path as well?

Reply:
@dschaller That's a good idea, I extended it to a take a string which will load an img from a given path, let me know if that's what you were thinking. I also updated the description with an example